configure.ac

```diff
@@ -38,6 +38,13 @@ AC_ARG_ENABLE(
 	[enable_distonly="no"]
 )
 
+AC_ARG_ENABLE(
+	[password-save],
+	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
+	[enable_password_save="yes"],
+	[enable_password_save="no"]
+)
```

Comment:
I tried it with "yes" in the last arg and it does not work as expected.
on the other side, https://travis-ci.org/OpenVPN/openvpn-gui/builds/148929302 works exactly as expected


I agree that we can omit default value. however, it is not clear about yes/no choice